src/battle/move/unknown_move/779C90.c

```diff
@@ -1,7 +1,34 @@
 #include "common.h"
 
+#include "battle/battle.h"
+
 #define NAMESPACE battle_move_unknown_move
 
 #include "world/common/UnkMoveFunc1.inc.c"
 
-INCLUDE_ASM(s32, "battle/move/unknown_move/779C90", func_802A10E4_779D74);
+extern s32 D_802A26D0_77B360;
+
+ApiStatus func_802A10E4_779D74(Evt* script, s32 isInitialCall) {
+    Actor *enemyTargetActor;
+    u32 enemyTargetPowerBounceChance;
+
+    enemyTargetActor = get_actor((s32) get_actor(script->owner1.enemyID)->targetActorID);
+
+    script->varTable[0] = 99;
+
+    enemyTargetPowerBounceChance = enemyTargetActor->actorBlueprint->powerBounceChance;
+
+    if (is_ability_active(0) != 0) {
+        enemyTargetPowerBounceChance += 7;
+    }
+
+    D_802A26D0_77B360 *= enemyTargetPowerBounceChance;
```

Comment:
Is the / 100 required on a separate line, or could the entire calculation be on this line only?

Reply:
Also, I guess this global would be chance as percentage, though maybe it's used elsewhere

Reply:
It seems like it does need to be on a separate line, although I could just be missing something.

Reply:
Globals are a bit weird, so it probably is correct as you have it, but I figured it was worth a shot :)

Reply:
:+1: